proto/ocsf_ext/finding_info/v1/finding_info.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package smithy.security.smithy.ocsf_ext.finding_info.v1;
+
+option go_package = "github.com/smithy-security/smithy/ocsf_ext/finding_info/v1";
+
+// DataSource is used to define Data Sources described on https://schema.ocsf.io/1.3.0/objects/finding_info.
+message DataSource {
+  // TargetType specifies the target type.
+  enum TargetType {
+    TARGET_TYPE_UNSPECIFIED = 0;
+    TARGET_TYPE_REPOSITORY = 1;
+  }
+
+  // URISchema specifies the URI schema.
+  // For example:
+  // - purl: pkg:npm/%40angular/animation@12.3.1" -> "pkg"
+  // - file: file://main.go -> "file"
+  enum URISchema {
+    URI_SCHEMA_UNSPECIFIED = 0;
+    URI_SCHEMA_FILE = 1;
+    URI_SCHEMA_PURL = 2;
+  }
+
+  // URI specifies uri content.
+  message URI {
```

Comment:
There are no commonly available types to be used here so this should work ok for us.